dll/win32/browseui/lang/pt-PT.rc

```diff
@@ -226,7 +231,7 @@ END
 STRINGTABLE
 BEGIN
     9250 "Mostra informações sobre o programa, o número da versão e os direitos autorais."
-    9252 "Mostra infomrações para depuração."
+    9252 "Mostra informações para depuração."
```

Comment:
This is good 2/2. Aside those 2 changed lines, I can't spot anything that would really improve the translation.

Reply:
Are you a native speaker so that you can judge whether a native person has improved the translation in his own language or not?

Reply:
No, but the previous translators were.
And I just expressed my thoughts here, I didn't actively reject anything.